docs/how-to/extract-a-backup.md

```diff
@@ -45,6 +45,22 @@ in the right place before running the command—or see below about the
 `--destination` flag.
 
 
+## Extract the configuration file used to create an archive
+
+<span class="minilink minilink-addedin">New in version 1.7.15</span> Borgmatic
+also stores the configuration file used to create an archive, inside the
+archive itself. This is useful in cases where you've lost your configuration
+file, or you want to see what configuration was used to create a particular
+archive.
+
+To extract the configuration file from an archive, use the `config bootstrap` action:
+
+```bash 
+borgmatic config bootstrap --repository repo.borg --destination /tmp
+```
+
+This extracts the configuration file from the latest archive in the repository `repo.borg` to `/tmp/config.yaml`.
```

Comment:
Wouldn't it be more like `/tmp/etc/borgmatic/config.yaml`? (You could explicitly mention that it includes the directory components of its original path or just let the example speak for itself.) Also, this last sentence should probably start with "For instance," or similar, as the user may very well have different configuration file(s) with different paths.